internal/stdlib/os/exec/exec.go

```diff
@@ -0,0 +1,789 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package exec runs external commands. It wraps os.StartProcess to make it
+// easier to remap stdin and stdout, connect I/O with pipes, and do other
+// adjustments.
+//
+// Unlike the "system" library call from C and other languages, the
+// os/exec package intentionally does not invoke the system shell and
+// does not expand any glob patterns or handle other expansions,
+// pipelines, or redirections typically done by shells. The package
+// behaves more like C's "exec" family of functions. To expand glob
+// patterns, either call the shell directly, taking care to escape any
+// dangerous input, or use the path/filepath package's Glob function.
+// To expand environment variables, use package os's ExpandEnv.
+//
+// Note that the examples in this package assume a Unix system.
+// They may not run on Windows, and they do not run in the Go Playground
+// used by golang.org and godoc.org.
+package exec
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"internal/syscall/execenv
```

Comment:
I am not sure what is going on here. Why is the ending quote missing here?

Reply:
If you're confused I'm doubly so. Possibly a backspace by accident? The first commit doesn't build at ALL as it's using things that don't exist until the second one comes around and fixes things up, so maybe my linter/editor was trying to get rid of this import that didn't exist and I kept having to manually re-add it and forgot the ending quote? I'm just spit-balling, I don't remember

Reply:
No worries, it gets removed in the next commit anyway. So this is fine.